test/components/project/home-block.spec.js

```diff
@@ -125,4 +125,19 @@ describe('ProjectHomeBlock', () => {
     const rows = block.findAllComponents(FormRow);
     rows.map((row) => row.props().form.name).should.eql(['a', 'c', 'd', 'e']);
   });
+
+  it('shows form ID in parentheses next to duplicated form names', () => {
+    const project = testData.extendedProjects.createPast(1).last();
+    testData.extendedForms.createPast(1, { name: 'Same Name', xmlFormId: 'a' });
+    testData.extendedForms.createPast(1, { name: 'Same Name', xmlFormId: 'b' });
```

Comment:
Is this check case sensitive? If so, the test is fine. If not, it might be good to compare "Same Name" and "same name".

Reply:
Ahh good idea. It's case-insensitive now!